src/game/bosses/boss_4.c

```diff
@@ -359,7 +359,7 @@ static void sub_8041A08(AeroEgg *boss)
         Player *p = &gPlayer;
 
         if ((p->speedAirY > 0) && (p->moveState & MOVESTATE_IN_AIR)) {
-            if (!sub_800DF38(s, worldX, worldY, p) == COLL_NONE) {
+            if ((!sub_800DF38(s, worldX, worldY, p)) == COLL_NONE) {
```

Comment:
If that was due to the compiler warning, wouldn't it still work putting the brace _after_ the `!`?
To me it'd make more sense.